shared/core/types/proto/relay.proto

```diff
@@ -6,24 +6,37 @@ option go_package = "github.com/pokt-network/pocket/shared/core/types";
 
 // TECHDEBT: These structures were copied as placeholders from v0 and need to be updated to reflect changes in v1
 message Relay {
-    RelayPayload payload = 1;
-    RelayMeta meta = 2;
+    RelayMeta meta = 1;
+    oneof RelayPayload {
```

Comment:
Should we `snake_case` `RelayPayload` to stay consistent with other proto fields? I believe it gets auto camel-cased when converted to a go struct.